src/wallet/wallet.cpp

```diff
@@ -3252,9 +3252,6 @@ bool ReserveDestination::GetReservedDestination(const OutputType type, CTxDestin
         return false;
     }
 
-    if (!pwallet->CanGetAddresses(internal)) {
```

Comment:
In commit "Key pool: Move CanGetAddresses call" (f034e5748aa2fac9fb8c9010c806d00b2b8d79ba)

Maybe say something like "Call LegacyScriptPubKeyMan::CanGetAddresses directly instead of calling CWallet::CanGetAddresses to only query the relevant key manager" in the commit description be clearer about what the change is supposed to do.

In case it helps anybody else, after 9b161a718f3c9d301419940cc9456f2065f8ca58 from #17261 there can more than keymanager and CWallet::CanGetAddresses will return false any of them can't generate addresses, so it wouldn't be right to call that method here.

Reply:
Done